src/forms/Select/Select.module.css

```diff
@@ -0,0 +1,122 @@
+/*
+ * Default Select styles
+ */
+.Select {
+  appearance: none;
+  display: flex;
+  color: var(--brand-color-text-default);
+  background-color: var(--brand-color-canvas-default);
+  border: none;
+  border-radius: var(--brand-control-border-radius);
+  line-height: var(--brand-text-lineHeight-200);
+  transition: var(--brand-control-animation-duration) var(--brand-control-animation-easing);
+  transition-property: color, background-color, box-shadow, border-color;
+  width: 100%;
+}
+
+@media screen and (forced-colors: active) {
+  /* Maintain dark bg color in Firefox on Windows high-contrast mode */
+  .Select:disabled {
+    background-color: -moz-combobox;
+  }
+}
+
+.Select::placeholder {
+  color: var(--brand-color-text-subtle);
+  opacity: 1;
+}
+
+.Select:focus-visible {
+  border-color: var(--brand-color-focus);
+  outline: none;
+  box-shadow: inset 0 0 0 1px var(--brand-color-focus);
+}
+
+.Select[disabled] {
+  color: var(--brand-color-text-muted);
+  cursor: not-allowed;
+  background-color: var(--base-color-scale-gray-1); /* FIXME: with text input bg token */
+  border-color: var(--brand-color-border-default);
```

Comment:
Disabled styles should be:

| Property | Light | Dark |
|--------|--------|--------|
| `border-color` | gray.2 | gray.5 |
| `background` | gray.1 | gray.8 |
| `color` (text) | gray.4 | gray.5 |